plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx

```diff
@@ -105,16 +107,24 @@ export const KubernetesContent = ({ entity }: KubernetesContentProps) => {
   >(undefined);
   const [error, setError] = useState<string | undefined>(undefined);
 
+  const googleAuth = useApi(googleAuthApiRef);
   useEffect(() => {
-    kubernetesApi
-      .getObjectsByServiceId(entity.metadata.name)
-      .then(result => {
-        setKubernetesObjects(result);
-      })
-      .catch(e => {
-        setError(e.message);
-      });
-  }, [entity.metadata.name, kubernetesApi]);
+    (async () => {
+      // TODO: Query Kubernetes backend to get info on which auth providers are in use
```

Comment:
We may want to provide a frontend interface for k8s auth too, as it would become a pretty clean place to switch between different methods in an app. We'd likely need something of the sort anyway to be able to support custom auth implementations.

What I really wanna figure out is if there's some good common contracts that we can put in place both between the frontend and backend, as well as internally within the backend too